inst/include/interface/rcpp/rcpp_objects/rcpp_tmb_distribution.hpp

```diff
@@ -287,8 +287,7 @@ class DlnormDistributionsInterface : public DistributionsInterfaceBase {
  public:
   ParameterVector x;       /**< observation */
   ParameterVector expected_values; /**< mean of the distribution of log(x) */
-  ParameterVector log_logsd;   /**< log standard deviation of the distribution of log(x) */
-  Rcpp::String input_type; /**< character string indicating type of input: data, re, prior */
+  ParameterVector log_sd;   /**< log standard deviation of the distribution of log(x) */
```

Comment:
Can you please be a bit more verbose here. I was expecting something more like "the natural logarithm of the standard deviation of the distribution of log(x), i.e., log(sd(log(x)) in R code. The natural log of the standard deviation is necessary because ..." where you will need to fill out the ellipse with actual relevant text.

Reply:
[thought]: I am still not convinced that we should be using log_sd instead of something more generic that will fit more distributions that the normal and lognormal. But, I am fine with keeping it for now.